src/sidebar/templates/top-bar.html

```diff
@@ -49,6 +49,7 @@
     <a class="top-bar__btn"
        ng-click="vm.onSharePage()"
        ng-if="vm.showSharePageButton()"
+       aria-label="Share this page"
```

Comment:
Do we need an `aria-label` _and_ a `title` for this button? What effects does that have for a screen reader?

Reply:
`title` provides the tooltip on over, but does not reliably communicate to a screenreader. Without the `aria-label`, a screenreader will announce this simply as "link." 

We don't _need_ both, but since the other elements in this toolbar have `titles` (and therefore tooltips on hover) we probably want to keep both for consistency's sake.